component-runtime-beam/src/main/java/org/talend/sdk/component/runtime/beam/spi/record/AvroRecord.java

```diff
@@ -81,7 +81,7 @@ public AvroRecord(final Record record) {
 
                             if (avroValue != null) {
                                 final org.apache.avro.Schema.Field field =
-                                        this.schema.getDelegate().getField(sanitizeConnectionName(entry.getName()));
+                                        this.schema.getActualDelegate().getField(sanitizeConnectionName(entry.getName()));
```

Comment:
Should not we also put raw/original name in the according avro property ?

Reply:
Here, we don't create a field, we get an existing one, so original name should have been already saved.
Here, "normally"; entry.getName is already sanitized, and so sanitizeConnectionName may be useless, but generate no bug; so it's for another JIRA.

Reply:
Apparently, we're also loosing all properties...

Reply:
Fixed for schema properties (and tested for both)